prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -20,7 +20,10 @@ presubmits:
         - -c
         - >-
           cd /home/prow/go/src/k8s.io/test-infra &&
-          bazel run prow/cmd/checkconfig -- --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
+          bazel run prow/cmd/checkconfig
+          --
+          --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
+          --job-config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/prowjobs
```

Comment:
I don't think this needs to use `bash -c` and these paths can replace `/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra` with `.` since that will be the working dir when the job starts.

Reply:
hummm, but the bazel rule is under k8s/test-infra, I didn't find a way to execute bazel rules under a different workspace?

Reply:
Oh I see.
/shrug

Reply:
Maybe we can publish an official checkconfig image for other repos (like here) to use :-)

Reply:
https://gcr.io/k8s-prow/checkconfig

Reply:
ahhh ^^ thanks